prometheus-to-sd/translator/metrics.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2018 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package translator
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	componentMetricsAvailability = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "component_metrics_availability",
+			Help: "Contains true(1.0) if /metrics endpoint of the component is available, otherwise false(0.0)",
+		},
+		[]string{"component"},
+	)
+
+	timeseriesesPushed = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "timeserieses_pushed",
```

Comment:
I think that plural for `time series` is also `time series`, also this is a counter, so probably should end in `_total` by convention

Reply:
Thanks, I have renamed as it was suggested.